src/utils/dayViewLayout.js

```diff
@@ -231,6 +231,9 @@ export default function getStyledEvents ({
   let styledEvents = []
   // idx of top-most, left-most event in each group of events
   let idx = 0
+  let prevNbrOfColumns = 0;
+  let childZIndex = 3;
```

Comment:
This is a magic number and should be explained more in the code. E.g. by extracting it to a clearly named const or with a code comment.

Reply:
Also, is this ever reassigned? If not, we should make this `const`.